packages/api/src/models/locationModel.js

```diff
@@ -190,15 +190,6 @@ class Location extends baseModel {
       },
     };
   }
-
-  static async isCropEnabled(locationId) {
-    const location = await this.query().whereNotDeleted().findById(locationId)
-      .withGraphJoined(`[
-           field, garden, greenhouse
-        ]`);
-
-    return location?.field || location?.garden || location?.greenhouse;
-  }
```

Comment:
I don't think we should complicate objectionModels

Reply:
Why not? MVC puts data and application logic primarily in models. I'd like to see future LF codebase eliminate database manipulation from controllers.